modules/serverless-service-mesh-resources.adoc

```diff
@@ -28,19 +24,15 @@ spec:
     hosts:
     - "*"
 ----
-
-.. Apply the YAML file:
+<1> The namespace where your Knative services have been created. This namespace must be a member of the Service Mesh member roll.
```

Comment:
Is _This namespace must be a member of the Service Mesh member roll._ essentially the same as _A namespace that is part of the Service Mesh member roll._? I like the latter wording better. Seems simpler.

Reply:
Not really, something could be a namespace that's in the SMMR without being the namespace where the ksvcs exist, so I think it's clearer to state the full thing.